packages/core/src/anchor/anchor-processing-loop.ts

```diff
@@ -39,46 +41,54 @@ export class AnchorProcessingLoop {
       'anchorRequestAge',
       METRICS_REPORTING_INTERVAL_MS
     )
-    this.#loop = new ProcessingLoop(logger, store.infiniteList(batchSize), async (streamId) => {
-      try {
-        logger.verbose(
-          `Loading pending anchor metadata for Stream ${streamId} from AnchorRequestStore`
-        )
-        const entry = await store.load(streamId)
-        const event = await cas.getStatusForRequest(streamId, entry.cid).catch(async (error) => {
-          logger.warn(`No request present on CAS for ${entry.cid} of ${streamId}: ${error}`)
-          const requestCAR = await eventHandler.buildRequestCar(streamId, entry.cid)
-          return cas.create(new AnchorRequestCarFileReader(requestCAR))
-        })
-        const isTerminal = await eventHandler.handle(event)
-        logger.verbose(
-          `Anchor event with status ${event.status} for commit CID ${entry.cid} of Stream ${streamId} handled successfully`
-        )
-        if (isTerminal) {
-          // Record the latency of how long this particular anchor request existed in the
-          // AnchorRequestStore.  The "record" function will compare the current time to the
-          // "timestamp" field from "entry", which was set as the current time when the request was
-          // first written into the AnchorRequestStore.
-          this.#anchorPollingMetrics.record(entry)
-          // Remove iff tip stored equals to the tip we processed
-          // Sort of Compare-and-Swap.
-          await this.#anchorStoreQueue.run(streamId.toString(), async () => {
-            const loaded = await store.load(streamId)
-            if (loaded.cid.equals(entry.cid)) {
-              await store.remove(streamId)
-            }
+
+    const concurrency =
+      Number(process.env.CERAMIC_ANCHOR_POLLING_CONCURRENCY) || DEFAULT_CONCURRENCY
+    this.#loop = new ProcessingLoop(
+      logger,
+      concurrency,
```

Comment:
this looks like a big diff but it's mostly formatting.  Literally the only change here is passing the `concurrency` arg to the `ProcessingLoop` constructor.